lib/platform/github/index.ts

```diff
@@ -1664,3 +1676,37 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
   }
   return alerts;
 }
+
+export async function pushFiles({
+  branchName,
+  files,
+  message,
+}: git.CommitFilesConfig): Promise<git.CommitSha | null> {
+  const additions = files.map(({ name: path, contents }) => ({
+    path,
+    contents: Buffer.from(contents).toString('base64'),
+  }));
+  const fileChanges = { additions };
+
+  let result = null;
+  try {
+    const variables = {
+      repo: config.repository,
+      repositoryId: config.repositoryId,
+      branchName: `refs/heads/${branchName}`,
+      oid: config.defaultBranchOid,
+      fileChanges,
+      message,
+    };
+
+    const commitRes = await githubApi.requestGraphql<{
+      createCommitOnBranch: { commit: { oid: string } };
+    }>(commitFilesMutation, { variables });
+
+    result = commitRes.data?.createCommitOnBranch?.commit?.oid;
+  } catch (err) {
+    logger.debug({ err });
+  }
+
+  return result ?? null;
```

Comment:
Should we throw or silently return `null` instead?
The only bad scenario I can imagine here is if branch already exists. But for this case, it will be reused and this code wouldn't be called at all.